### ISSUE TITLE
Console error logs

### ISSUE THREAD

**Issue by antgly** [vknabel/vscode-swift-development-environment#18]
I think it would be great if we could post our error logs somewhere. There's no way that I'm going to pay ~ $20 a month for AppCode.

**Comment by antgly** [comment]
I get multiple since I've had this extension installed even when I'm getting desired output. My point is that Xcode sucks and we need the robustness that Microsoft's (am I dreaming?) VSCode provides.

**Comment by vknabel** [comment]
Yes. Currently this extension logs too much and particularly irrelevant information.
My current goal for this project is to get it more stable and easier to use. I think this is definitely part of it. 

You can just add comments to this issue (if everything still works) or create new ones (if something does not work). If you wish some more features, you can just create an issue.
